.github/workflows/publish.yml

```diff
@@ -22,4 +22,20 @@ jobs:
         - name: Use Node.js
           uses: actions/setup-node@v1
         - name: publish to master
-          run: bash ./scripts/publish.sh        
\ No newline at end of file
+          run: COMMIT_MESSAGE=$(git log --format=%B -n 1) &&
+            if [[ $COMMIT_MESSAGE == "release-"* || $COMMIT_MESSAGE == "[Simsala] automatic release"* ]];
+            then
+              echo "Publishing" &&
+              git config pull.rebase false &&
+              git config user.email "bot@lunie.io" &&
+              git config user.name "Publish Lunie Bot" &&
+              git remote add bot https://${GIT_BOT_TOKEN}@github.com/luniehq/lunie.git &&
+              git checkout master &&
+              git pull &&
+              git merge --no-edit origin/develop &&
+              git push;
```

Comment:
do we need the `&&`?

Reply:
yes, I tried it. It is because here it is not a script but directly command line